sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -866,7 +866,7 @@ object SQLConf {
       .createWithDefault(true)
 
   val FAIL_AMBIGUOUS_SELF_JOIN =
```

Comment:
Also, `FAIL_AMBIGUOUS_SELF_JOIN` -> `FAIL_AMBIGUOUS_SELF_JOIN_ENABLED` because `_ENABLED` patter is more popular for `.enabled` conf.